app/models/report.rb

```diff
@@ -0,0 +1,92 @@
+class Report
+  delegate :url_helpers, :to => 'Rails.application.routes'
```

Comment:
Code Climate/Rubocop have an appropriate suggestion to use the 1.9+ Hash syntax here.